data/sql/nkg.sql

```diff
@@ -693,7 +693,75 @@ INSERT INTO "usage" VALUES (
     'NKG',  'SCOPE_GENERIC'     -- scope
 );
 
+-- NO
+INSERT INTO "helmert_transformation" VALUES (
+    'NKG','PAR_2020_NO', -- operation auth+code
+    'NKG_ETRF14 to ETRF93@2000.0', -- name
+    'Transformation from NKG_ETRF14 to ETRF93, at transformation reference epoch 2000.0', -- description / remark
+    'EPSG','1033',  -- method auth+code
+    'Position Vector transformation (geocentric domain)',
+    'NKG','ETRF14', -- source auth+code
+    'EPSG','7922',  -- target auth+code
+    0.01,           -- accuracy
+    -0.05172,       -- x
+    0.13747,        -- y
+    -0.01648,       -- z
+    'EPSG','9001',
+    0.00268452,     -- rx
+    0.00329165,     -- ry
+    -0.00116569,    -- rz
+    'EPSG','9104',
+    0.002583,       -- s
+    'EPSG','9202',
+    NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,
+    'NKG 2020',     -- operation version
+    0
+);
 
+INSERT INTO "usage" VALUES (
+    'NKG', '5041',              -- usage auth+code
+    'helmert_transformation',   -- object_table_name
+    'NKG','PAR_2020_NO',        -- object auth+code
+    'EPSG', '1352',             -- extent: Norway - onshore
+    'NKG', 'SCOPE_GENERIC'      -- scope
+);
+
+INSERT INTO "other_transformation" (
+    auth_name,
+    code,
+    name,
+    description,
+    method_auth_name,
+    method_code,
+    method_name,
+    source_crs_auth_name,
+    source_crs_code,
+    target_crs_auth_name,
+    target_crs_code,
+    accuracy,
+    operation_version,
+    deprecated
+)
+VALUES(
+    'NKG', 'NKG_ETRF14_ETRF93_2000', -- object auth+code
+    'NKG_ETRF14 to ETRF93@2000.0', -- name
+    'Transformation from NKG_ETRF14 to ETRF93, at transformation reference epoch 2000.0', -- description / remark
+    'PROJ', 'PROJString', 
+    '+proj=xyzgridshift +grids=no_kv_NKGETRF14_EPSG7922_2000.tif',
```

Comment:
that grid should likely be registered in data/sql/grid_alternatives.sql

Reply:
@kbevers Should we add NKG resource files into grid_alternatives.sql?

Reply:
Doesn't hurt to do so. I am not entirely sure what is gained by doing it, as far as I know everything works as intended without it being there.

Reply:
> I am not entirely sure what is gained by doing it,

The grid would then be referenced as available from the CDN

Reply:
The NKG velocity grids 'eur_nkg_nkgrf03vel_realigned.tif' and 'eur_nkg_nkgrf17vel.tif' are not yet added to grid_alternatives. I tried, but I could not find any named 'proj_method' definition for geodetic velocity grids.

Reply:
> I tried, but I could not find any named 'proj_method' definition for geodetic velocity grids.

You may need to add 'velocity_grid' in the enumeration of the constraint check_grid_alternatives_proj_method of the CREATE TABLE grid_alternatives definition in proj_db_table_defs.sql